release-notes/README.md

```diff
@@ -4,7 +4,23 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
-## March 3, 2021
```

Comment:
Nice catch, leaving in the past 🙈 
Can you please update the year for this one too https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/blob/master/using-mobile-extensions/adobe-analytics/adobe-analytics-release-notes.md#march-3-2021 ?

Reply:
Have updated it already :)